rs-bao/src/components/common/CoachBadge.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <p class="badge" :class="color">
+    {{ text }}
+  </p>
+</template>
+
+<script>
+export default {
+  props: ["color", "title"],
+  computed: {
+    text() {
```

Comment:
viết thế này cũng được nhưng nên dùng filter nhé.
https://vuejs.org/v2/guide/filters.html

Reply:
Dạ anh Cường ơi, em có đọc trong docs vue3 thì nó ghi : Filters are removed from Vue 3.0 and no longer supported.
Link docs anh: https://v3.vuejs.org/guide/migration/filters.html

Reply:
Do em đang làm Vue3 á anh ~~

Reply:
@baoluonghoang  à thế thì okila